github/admin_orgs.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import "context"
+
+// createOrgRequest is a subset of Organization and is used internally
+// by Create to pass only the known fields for the endpoint
+//
+type createOrgRequest struct {
+	Login *string `json:"login"`
```

Comment:
If these two fields are required, you can get rid of the pointer.  Otherwise, please add `omitempty` to the JSON tag.
For example:

```go
`json:"login,omitempty"`
```